sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_path_client.py

```diff
@@ -204,6 +218,28 @@ def _create(self, resource_type, content_settings=None, metadata=None, **kwargs)
             For example, if p is 0777 and u is 0057, then the resulting permission is 0720.
             The default permission is 0777 for a directory and 0666 for a file. The default umask is 0027.
             The umask must be specified in 4-digit octal notation (e.g. 0766).
+        :keyword str owner:
+            The owner of the blob or directory. Default value is None.
+        :keyword str group:
+            The owning group of the blob or directory. Default value is None.
+        :keyword str acl:
+            Sets POSIX access control rights on files and directories. The value is a
+            comma-separated list of access control entries. Each access control entry (ACE) consists of a
+            scope, a type, a user or group identifier, and permissions in the format
+            "[scope:][type]:[id]:[permissions]". Default value is None.
+        :keyword str lease_id:
+            Lease ID, in a GUID string format. The Blob service returns
+            400 (Invalid request) if the lease ID is not in the correct format. See Guid
+            Constructor (String) for a list of valid GUID string formats. Default value is None.
+        :keyword int lease_duration:
+            The lease duration is required to acquire a lease, and specifies the
+            duration of the lease in seconds.  The lease duration must be between 15 and 60 seconds or -1
+            for infinite lease. Default value is None.
+        :keyword expiry_options:
```

Comment:
Two things for `expiry_options`. The type should just be `str` as we don't export the generated model `PathExpiryOptions` and this matches what we do in `set_file_expiry`. We could also consider the `Literal[]` type here as we did [here](https://github.com/Azure/azure-sdk-for-python/blob/01108413a17bfc9f75c2c571fbb6799f1cce6867/sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py#L2026)

Also, we should update the description to include the possible values for this `str`. I think matching the `set_file_expiry` description looks good. Though I'm not sure we actually support all those options? I remember Sean mentioning something about this so maybe check what .NET supports here.

"Indicates mode of the expiry time.
Possible values include: 'NeverExpire', 'RelativeToCreation', 'RelativeToNow', 'Absolute'"

Reply:
I think it may be we only support RelativeToCreation or RelativeToNow, not both, since they will be the same.

Reply:
I think it may be we only support RelativeToCreation or RelativeToNow, not both, since they will be the same.